function/requirements.txt

```diff
@@ -13,20 +13,27 @@
 # want to install the CPU or GPU version of TensorFlow.
 #
 ################################################################
+# Wheel needs to be installed directly for tensorflow installation to work
```

Comment:
These changes may break the regular x86_64 version, what if we added a new requirements-armhf.txt file?

Reply:
ah yes good catch. we'll need that